portfolio/src/main/webapp/index.html

```diff
@@ -41,6 +41,14 @@
                         <img class="d-block h-100" src="./images/tenniscourt.jpg" alt="Third slide">
                     </div>
                 </div>
+                <a class="carousel-control-prev" href="#carouselExampleControls" role="button" data-slide="prev" onclick="$('#hobbySlides').carousel('prev')">
+                    <span class="carousel-control-prev-icon" aria-hidden="true"></span>
+                    <span class="sr-only">Previous</span>
+                </a>
+                <a class="carousel-control-next" href="#carouselExampleControls" role="button" data-slide="next" onclick="$('#hobbySlides').carousel('next')">
+                    <span class="carousel-control-next-icon" aria-hidden="true"></span>
+                    <span class="sr-only">Next</span>
+                </a>
             </div>
             <p>
                 Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras ac lacinia velit. Vivamus tristique lacinia nibh, sed tristique sem. Etiam ultrices sapien quis faucibus malesuada. Morbi hendrerit ut diam sit amet dictum. Aenean turpis nisl, fringilla convallis justo varius, blandit vehicula arcu. Sed viverra elementum odio. Nulla at eros sed massa eleifend pellentesque. Ut dignissim nisl ut viverra dictum. Curabitur id lectus viverra, feugiat libero eget, consequat tortor.
```

Comment:
You will add the content in another comment?

Reply:
Looks good now. Yeah having buttons is more intuitive.

Reply:
Sure, I'm adding, just added contact data. At the end I will provide some more meaningful text about myself and my hobby, for now only example content to see how it looks like.